Apeswap-Bots/detect-large-banana-token-mints/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "apeswap-1",
+  "version": "0.0.1",
+  "description": "Detects large BANANA mints",
+  "chainIds": [
+    56
+  ],
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "format": "prettier --write \"src/**/*.ts\"",
+    "test": "jest"
+  },
+  "dependencies": {
+    "forta-agent": "^0.1.3"
+  },
+  "devDependencies": {
+    "@types/jest": "^27.0.1",
+    "@types/nodemon": "^1.19.0",
+    "forta-agent-tools": "^2.0.5",
+    "jest": "^27.0.6",
+    "nodemon": "^2.0.8",
+    "prettier": "^2.6.2",
+    "ts-jest": "^27.0.3",
+    "typescript": "^4.3.4"
+  },
+  "repository": "https://github.com/NethermindEth/Forta-Agents/tree/%40apeswap/large-banana-mint/Apeswap-Bots/detect-large-banana-token-mints"
```

```suggestion
  "repository": "https://github.com/NethermindEth/Forta-Agents/tree/main/Apeswap-Bots/detect-large-banana-token-mints"
```
(This will be the final url)